www/src/data/tags-docs.js

```diff
@@ -20,6 +20,7 @@ export const TAGS_AND_DOCS = new Map([
   [`cdn`, ``],
   [`cli`, ``],
   [`client-side-routing`, `/docs/client-only-routes-and-user-authentication`],
+  [`cloudflare`, "/docs/deploying-to-cloudflare-workers"],
```

Comment:
```suggestion
```

Since there aren't any blog posts tagged Cloudflare we don't need to add this here 🙂